go.mod

```diff
@@ -45,7 +46,7 @@ require (
 	github.com/rogpeppe/fastuuid v1.2.0 // indirect
 	github.com/tv42/zbase32 v0.0.0-20160707012821-501572607d02
 	github.com/urfave/cli v1.18.0
-	golang.org/x/crypto v0.0.0-20200109152110-61a87790db17
+	golang.org/x/crypto v0.0.0-20200115085410-6d4e4cb37c7d
```

Comment:
Any way we can suppress this?

Reply:
Good catch!
It seems this is added automatically on running `go get ./...` because the new version of `btcsuite/btcutil` that contains the PSBT stuff references it. I added an explicit overwrite to prevent using a version that we haven't verified.